### PR TITLE
Minor fixes related to import.

### PR DESCRIPTION
**Pull request by nickygerritsen**
Found all these while working on the domlogo changes.

### CODE REVIEW

File: webapp/src/Service/DOMJudgeService.php [hunk 1]
```diff
@@ -654,7 +654,13 @@ public function internalApiRequest(string $url, string $method = Request::METHOD
             return null;
         }
 
-        return $this->jsonDecode($response->getContent());
+        $content = $response->getContent();
+
+        if ($content === '') {
+            return null;
```
Comment (meisterT) [inline webapp/src/Service/DOMJudgeService.php:660]: Why is this necessary?

Comment (nickygerritsen) [reply webapp/src/Service/DOMJudgeService.php:660]: The message of 33db981027a0c58fce7e9b4445d6410d06ded8dd explains this, but the API returns an empty response for 204's, like when uploading an image. This can't be JSON decoded and apparently breaks the import script.

**Review by meisterT — APPROVED** [review]
